synapse/tests/test_lib_types.py

```diff
@@ -546,18 +547,14 @@ def test_type_xref(self):
             self.raises(BadTypeValu, core.getTypeNorm, 'foo:bar', ['oh', 'my', 'its', 'broked'])
             self.raises(BadInfoValu, core.addType, 'foo:baz', subof='xref', source='ou=org')
 
-    def test_types_isguid(self):
```

Comment:
since we're making sure a regex does what it expects to do here, why not move these tests into the test_types_guid so we know that guy is always going to be correct

Reply:
my thoughts were that the code that tests it directly should be in a test module named for the location of the code being tested.  no strong feelings either way though.